docs/metrics/exemplars/README.md

```diff
@@ -50,15 +54,15 @@ directory and run the following command:
 dotnet run
 ```
 
-The above would start the application at port 5000. Navigate to
+The above command would start the application at port 5000. Navigate to
```

Comment:
Maybe something like "Now the application is running, navigate to ... (the actual URL)"

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4195